lib/Service/OpenProjectAPIService.php

```diff
@@ -319,11 +320,41 @@ public function getOpenProjectAvatar(
 			$response = $this->rawRequest(
 				$accessToken, $openprojectUrl, 'users/'.$openprojectUserId.'/avatar'
 			);
+			$headerType = $response->getHeader('Content-Type');
```

Comment:
```suggestion
			$imageMimeType = $response->getHeader('Content-Type');
```
headerType does not make a bit of a sense here i guess